pkg/cluster/initialize.go

```diff
@@ -58,6 +61,31 @@ func (u *simpleUpgrader) Initialize(ctx context.Context, cs *api.OpenShiftManage
 	return b.CreateBlockBlobFromReader(bytes.NewReader(csj), nil)
 }
 
+func (u *simpleUpgrader) CreateStorageAccount(ctx context.Context, cs *api.OpenShiftManagedCluster, accountName string) error {
+	parameters := azstorage.AccountCreateParameters{
+		Sku: &azstorage.Sku{
+			Name: azstorage.StandardLRS,
+		},
+		Kind:     azstorage.Storage,
+		Location: &cs.Location,
+	}
+	if accountName == cs.Config.ConfigStorageAccount {
```

Comment:
roll the registry acct back into the arm template, make this function CreateConfigStorageAccount and get rid of the if

Reply:
done